appveyor.yml

```diff
@@ -6,7 +6,6 @@ skip_commits:
     - .ci/
     - .circleci/
     - .github/
-    - .gitattributes
```

Comment:
@romani Do you know how this file list is used? Is it looking only at HEAD or all commits in PR?

Reply:
Appveyor is not quick question in execution, such CI benefits a lot from such check, but side effects are possible.